common/app/common/dfp/DfpData.scala

```diff
@@ -252,6 +260,9 @@ case class GuLineItem(
 
   val highMerchandisingTargets: Seq[String] = targeting.customTargetSets.flatMap(_.highMerchandisingTargets).distinct
 
+  val liveBlogTopTargetedSections: Set[String] =
+    targeting.customTargetSets.flatMap(_.liveblogTopTargetedSections).distinct.toSet
```

Comment:
Why use a `Set` instead of a `Seq` here? Just thinking about consistency, especially as you already call `distinct`